recipes/festival/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set name = "festival" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/festival/archive/5db6ee949f4e4e9da25c87bd03b90e69f9011393.tar.gz
+  sha256: 9f0d0e4f0e91fca4430ee4a6ca4d3d24bac444f36b4f4e9653c6119aadfae13e
+
+build:
+  run_exports:
+    - {{ pin_subpackage('festival', max_pin='x.x') }}
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [linux]
+    - libgomp  # [linux]
+    - libestools-dev  # [linux]
+    - libestools-static  # [linux]  This is needed to satisfy Make, but not used
+
+  host:
+    - libestools  # [linux]
+  run:
+    - libestools  # [linux]
+
+outputs:
+  - name: festival
+    script: install.sh  # [linux]
```

Comment:
Just FYI, conda also accepts a list of files with glob pattern matching instead of the name of an install script for separating outputs. It's less commonly used, but sometimes it's easier to implement especially if the artifacts are grouped in folders.

Reply:
The docs are not really verbose on how to specify where to put the files. I.e. I need to pick folder `src/include/festival` and put it into `$PREFIX/include/festival`. How do I specify that with the file lists?

Reply:
The paths are relative to $PREFIX. So if you want to include `$PREFIX/lib/libfestival.so` in the output, you would list `lib/libfestival*` or `lib/libfestival.so  # [linux]`

Reply:
The upstream package does provide a `make install` target. However, it's not very useful. It doesn't follow any standard FHS rules. Instead of patching the installer, I decided for the bunch of custom scripts which do the same work, but aren't compatible with the YAML install file lists.